packages/core/src/components/checkbox/checkbox.css

```diff
@@ -0,0 +1,188 @@
+/* import checkbox color variables */
+@import "hds-design-tokens/lib/color/component/checkbox.css";
+
+@svg-load icon url(../../svg/icon-check.svg);
+
+.hds-checkbox {
+  --size: 24px;
+  /* icon size relative to the checkbox size (--size) */
+  --icon-scale: 0.75;
+  --border-radius: 1px;
+  --border-width: 2px;
+  --outline-width: 3px;
+
+  --background-unselected: var(--color-background-checkbox-unselected);
+  --background-selected: var(--color-background-checkbox-selected);
+  --background-hover: var(--color-background-checkbox-hover);
+  --background-disabled: var(--color-background-checkbox-disabled);
+
+  --border-color-selected: var(--color-border-checkbox-selected);
+  --border-color-selected-hover: var(--color-border-checkbox-selected-hover);
+  --border-color-selected-focus: var(--color-border-checkbox-selected-focus);
+  --border-color-unselected: var(--color-border-checkbox-unselected);
+  --border-color-unselected-hover: var(--color-border-checkbox-unselected-hover);
+  --border-color-unselected-focus: var(--color-border-checkbox-unselected-focus);
+  --border-color-disabled: var(--color-border-checkbox-disabled);
+
+  --icon-color-unselected: var(--color-icon-checkbox-unselected);
+  --icon-color-selected: var(--color-icon-checkbox-selected);
+  --icon-color-disabled: var(--color-icon-checkbox-disabled);
+
+  --focus-outline-color: var(--color-focus-outline-checkbox-default);
+
+  --label-color: var(--color-label-checkbox-default);
+  --label-color-disabled: var(--color-label-checkbox-disabled);
+
+  align-items: center;
+  display: flex;
+  height: var(--size);
+  position: relative;
+}
+
+.hds-checkbox:not(:first-of-type) {
+  margin-top: var(--spacing-2-xs);
+}
+
+/* checkbox transitions */
+.hds-checkbox .hds-checkbox__input:hover::before,
+.hds-checkbox .hds-checkbox__input:focus::before,
+.hds-checkbox .hds-checkbox__input:hover + .hds-checkbox__label::after,
+.hds-checkbox .hds-checkbox__input:focus + .hds-checkbox__label::after {
+  transition: 85ms ease-out;
+  transition-property: background-color, border-color, transform;
+}
+
+.hds-checkbox .hds-checkbox__input:focus + .hds-checkbox__label::before {
+  transition: 85ms ease-out;
+  transition-property: box-shadow, transform;
+}
+
+/* NATIVE INPUT */
+.hds-checkbox .hds-checkbox__input {
+  appearance: none;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  height: var(--size);
+  left: 0;
+  outline: none;
+  position: absolute;
+  top: 0;
+  width: var(--size);
+}
```

Comment:
The cursor in not a pointer for the input:
![2020-05-11 15 38 33](https://user-images.githubusercontent.com/6904260/81562789-197d9d80-939e-11ea-9d97-1870014eda55.gif)

```suggestion
.hds-checkbox .hds-checkbox__input {
  appearance: none;
  -moz-appearance: none;
  -webkit-appearance: none;
  height: var(--size);
  left: 0;
  outline: none;
  position: absolute;
  top: 0;
  width: var(--size);
  cursor: pointer;
}
```

...or then bump the z-index for label, if that's intended to handle the cursor state.

Reply:
Whoopsie-daisy 😅